skimage/_shared/tests/test_coord.py

```diff
@@ -56,6 +57,18 @@ def test_ensure_spacing_batch_processing(p, size):
                           expected)
 
 
+@pytest.mark.skipif(
+    sys.platform != 'linux' or sys.version_info[:2] != (3, 9),
```

Comment:
this is still going to be problematic if we end up enabling tests for aarch64 or other emulated architecture later on.

I am not sure that this test case is not going to cause us more problems than it is worth

Reply:
I refactored this test in f631c6c to compare relative times rather than having an absolute timeout. On my system `batch_size=20000` is about 50% slower than `batch_size=2000`. Does this seem reasonable?